lib/dataTrigger/featureFlagsDataTriggers/createFeatureFlagDataTrigger.js

```diff
@@ -0,0 +1,21 @@
+const DataTriggerConditionError = require('../../errors/DataTriggerConditionError');
+const DataTriggerExecutionResult = require('../DataTriggerExecutionResult');
+
+async function createFeatureFlagDataTrigger(documentTransition, context, topLevelIdentity) {
```

Comment:
Where are you passing `topLevelIdentity`? I thought you will compare it with data contract owner id.